### ISSUE TITLE
A screen reader (e.g., NVDA) doesn't announce a question's description

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#7881]
T16900 - Accessibility issues
https://surveyjs.answerdesk.io/internal/ticket/details/T16900
***
A screen reader (NVDA, VoiceOver) does not read a question's description. Only a title text is pronounced.
![image](https://github.com/surveyjs/survey-library/assets/22372972/136018ae-dc75-4c6b-84b8-6c4c27a3d104)
[View Demo](https://plnkr.co/edit/2HHupvzu9Su91Gd3)
